src/aiohomeconnect/client.py

```diff
@@ -297,9 +296,7 @@ async def start_program(
         ha_id: str,
         *,
         program_key: ProgramKey,
-        name: str | None = None,
```

Comment:
Name is part of the example in the docs.

Reply:
Oh, that's true, sorry for the mistake. Fixing it